src/com/b44t/messenger/DcContact.java

```diff
@@ -3,7 +3,8 @@
 public class DcContact {
 
     public final static int DC_CONTACT_ID_SELF               = 1;
-    public final static int DC_CONTACT_ID_DEVICE             = 2;
+    public final static int DC_CONTACT_ID_INFO               = 2;
+    public final static int DC_CONTACT_ID_DEVICE             = 5;
```

Comment:
we changed this in the core and forgot to adapt this in android. was no issue in the past as DC_CONTACT_ID_DEVICE was just never used in android up to now.